components/playground/main.go

```diff
@@ -147,7 +154,7 @@ func hasDashboard(pdAddr string) bool {
 	return false
 }
 
-func bootCluster(version string, pdNum, tidbNum, tikvNum int, host string, monitor bool) error {
+func bootCluster(version, pdConfigPath, dbConfigPath, kvConfigPath string, pdNum, tidbNum, tikvNum int, host string, monitor bool) error {
```

Comment:
I think we should make further handling for the `pdConfigPath/dbConfigPath/kvConfigPath` and convert it to an absolute path. 
According to https://github.com/pingcap-incubator/tiup/blob/e08d28e360c6aee5044a9308c74eb994281be4e4/cmd/run.go#L198. Them will be related to the playground work directory, which is not same as the working directory of the user running `tiup playground`.

We can achieve our intention via the following code snippet.```
if !strings.HasPrefix(pdConfigPath, "/") && strings.HasPrefix(pdConfigPath, "~") {
    pdConfigPath = filepath.Join(os.Getenv(localdata.EnvNameWorkDir), pdConfigPath)
}
```

Reply:
@fredchenbj define a method `getAbsolutePath` in #82 , i think i can reuse that method here. But redefine that method here seems redundant and could cause conflicts. Maybe it is better to fix this problem after #82 merged so that i can use `getAbsolutePath` here. Is that OK?

Reply:
Yes, merge #82 first.